src/config.ts

```diff
@@ -189,15 +189,25 @@ export const RFQT_REGISTRY_PASSWORDS: string[] = _.isEmpty(process.env.RFQT_REGI
     ? []
     : assertEnvVarType('RFQT_REGISTRY_PASSWORDS', process.env.RFQT_REGISTRY_PASSWORDS, EnvVarType.JsonStringList);
 
-export const RFQT_API_KEY_WHITELIST: string[] = _.isEmpty(process.env.ALT_RFQ_MM_ENDPOINT)
+export const RFQT_API_KEY_WHITELIST: string[] = _.isEmpty(process.env.RFQT_API_KEY_WHITELIST)
```

Comment:
found typo, fixed \

Reply:
was this broke?

Reply:
no, they're defined so it doesn't hit this part of the code, but should be fixed anyway